man/chpl.txt

```diff
@@ -103,16 +103,18 @@ OPTIONS
   --[no-]dead-code-elimination   Enable [disable] dead code elimination.
 
   --fast            Turns off all runtime checks using --no-checks, turns
-                    on --no-ieee-float, -O and --specialize, and enables all
-                    compiler optimizations in the rest of this section.
+                    on -O and --specialize, and enables most other compiler
+                    optimizations described in this section (--no-ieee-float
+                    and --cache-remote are notable exceptions).
 
   --[no-]fast-followers   Enable [disable] the fast follower
                     optimization in which fast implementations of
                     followers will be invoked for specific leaders.
 
   --[no-]ieee-float   Disable [enable] optimizations that may affect IEEE
-                    floating point conformance. The default is --ieee-float,
-                    which ensures IEEE conformance.
+                    floating point conformance. The default is whatever level
+                    of optimization/IEEE floating point support your C compiler
```

Comment:
Could this be changed to say "back-end compiler" rather than "C compiler" (unless the statement doesn't refer to the LLVM compiler?  Also, on the next line, "with the C compiler optimization provided." seems a bit circular and vague.  This isn't completely satisfactory, but my similarly terse suggestion would be: "your back-end compiler provides at the given optimization level."